CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## [v0.11.0.0](https://github.com/LambdaHack/LambdaHack/compare/v0.10.3.0...v0.11.0.0)
 
+- Split the long Content.ModeKind texts into lines, [issue 283](https://github.com/LambdaHack/LambdaHack/issues/283)
```

Comment:
Thank you for thinking about the changelog. LambdaHack v0.11.0.0 has already been released. Please start a new section (v0.11.1.0) and move your item there.